apps/web/components/payroll/payroll-input.module.css

```
@@ -0,0 +1,44 @@
+.privacyBanner p { margin: 5px 0 0; color: var(--muted); font-size: .84rem; line-height: 1.5; }
+.lock { padding: 9px 10px; border: 1px solid rgba(117,230,204,.35); border-radius: 8px; color: var(--cyan); font-size: .65rem; font-weight: 900; letter-spacing: .13em; }
+.summaryGrid { display: grid; grid-template-columns: repeat(4, 1fr); gap: 1px; overflow: hidden; margin-top: 20px; border: 1px solid var(--line); border-radius: 16px; background: var(--line); }
+.summaryGrid article { min-width: 0; padding: 18px; background: var(--panel-strong); }
+.summaryGrid p, .summaryGrid span { margin: 0; color: var(--muted); font-size: .7rem; letter-spacing: .08em; text-transform: uppercase; }
+.summaryGrid strong { display: block; overflow: hidden; margin: 14px 0 6px; text-overflow: ellipsis; font-size: 1.16rem; }
+.good { color: var(--lime); }
+.warn { color: var(--warning); }
+.controlGrid { display: grid; grid-template-columns: 1fr 1fr; gap: 20px; margin-top: 20px; }
+.importActions { display: flex; gap: 10px; }
+.importActions > * { display: flex; flex: 1; align-items: center; justify-content: center; text-align: center; text-decoration: none; }
+.fileInput { position: absolute; width: 1px; height: 1px; overflow: hidden; clip: rect(0 0 0 0); clip-path: inset(50%); white-space: nowrap; }
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)